Integration test sometimes fails

The test "should be able to pull a file after trying to open it and it does not exist on disk and the no missing errors option is unset" failed on a CI run for #552 once, then succeeded on a re-run.

The run in question is [here](https://github.com/iterative/vscode-dvc/actions/runs/937056755), but it seems the old logs get wiped when the Action is re-run

I saved the first run's log just for this case, though the format looks a little nasty here.

```
2021-06-14T20:19:38.7709002Z [0m  1) Extension Test Suite
2021-06-14T20:19:38.7709812Z        TrackedExplorerTree
2021-06-14T20:19:38.7710992Z          should be able to pull a file after trying to open it and it does not exist on disk and the no missing errors option is unset:
2021-06-14T20:19:38.7712606Z [0m[31m     AssertionError: expected executeProcess to not have been called[0m[90m
2021-06-14T20:19:38.7714559Z   	at /home/runner/work/vscode-dvc/vscode-dvc/extension/dist/test/suite/fileSystem/views/trackedExplorerTree.test.js:137:42
2021-06-14T20:19:38.7715861Z   	at Generator.next (<anonymous>)
2021-06-14T20:19:38.7717048Z   	at fulfilled (dist/test/suite/fileSystem/views/trackedExplorerTree.test.js:24:58)
2021-06-14T20:19:38.7718422Z   	at processTicksAndRejections (internal/process/task_queues.js:93:5)
2021-06-14T20:19:38.7719464Z [0m
2021-06-14T20:19:38.7719859Z 
2021-06-14T20:19:38.7720459Z Error: 1 tests failed.
2021-06-14T20:19:38.7721799Z rejected promise not handled within 1 second: TypeError: Disposable.normalize is not a function or its return value is not iterable
2021-06-14T20:19:38.7723429Z stack trace: TypeError: Disposable.normalize is not a function or its return value is not iterable
2021-06-14T20:19:38.7725114Z 	at Disposer.track (/home/runner/work/vscode-dvc/vscode-dvc/node_modules/@hediet/std/disposable.js:97:36)
2021-06-14T20:19:38.7726652Z 	at /home/runner/work/vscode-dvc/vscode-dvc/extension/dist/extension.js:3356:28
2021-06-14T20:19:38.7727622Z 	at Array.forEach (<anonymous>)
2021-06-14T20:19:38.7729213Z 	at Extension.initializeRepositories (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/extension.js:3352:21)
2021-06-14T20:19:38.7731297Z 	at Extension.initialize (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/extension.js:3397:25)
2021-06-14T20:19:38.7732824Z 	at /home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup.js:929:24
2021-06-14T20:19:38.7734084Z 	at Generator.next (<anonymous>)
2021-06-14T20:19:38.7735365Z 	at fulfilled (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup.js:866:24)
2021-06-14T20:19:38.7736640Z 	at processTicksAndRejections (internal/process/task_queues.js:93:5)
2021-06-14T20:19:38.7737382Z 
2021-06-14T20:19:38.9159061Z Exit code:   1
2021-06-14T20:19:38.9159873Z Done
```

I think this is the relevant part of the massive log file for the whole run, but I can upload it we think there's some other valuable info.

--- a/extension/src/test/suite/fileSystem/views/trackedExplorerTree.test.ts
+++ b/extension/src/test/suite/fileSystem/views/trackedExplorerTree.test.ts
@@ -127,11 +127,17 @@ suite('Extension Test Suite', () => {
       const mockProcess = stub(Process, 'executeProcess').resolves(
         'M       non-existent.txt\n1 file modified'
       )
+      const pullAgs = {
+        args: ['pull', missingFile],
+        cwd: undefined,
+        env: process.env,
+        executable: 'dvc'
+      }
 
       await commands.executeCommand(openFileCommand, uri)
 
       expect(mockShowInformationMessage).to.be.calledOnce
-      expect(mockProcess).not.to.be.called
+      expect(mockProcess).not.to.be.calledWith(pullAgs)
 
       mockShowInformationMessage.resetHistory()
       mockShowInformationMessage.resolves(
@@ -142,12 +148,7 @@ suite('Extension Test Suite', () => {
 
       expect(mockShowInformationMessage).to.be.calledOnce
       expect(mockProcess).to.be.calledOnce
-      expect(mockProcess).to.be.calledWith({
-        args: ['pull', missingFile],
-        cwd: undefined,
-        env: process.env,
-        executable: 'dvc'
-      })
+      expect(mockProcess).to.be.calledWith(pullAgs)
 
       mockProcess.resetHistory()
       mockShowInformationMessage.resetHistory()
@@ -158,14 +159,14 @@ suite('Extension Test Suite', () => {
       await commands.executeCommand(openFileCommand, uri)
 
       expect(mockShowInformationMessage).to.be.calledOnce
-      expect(mockProcess).not.to.be.called
+      expect(mockProcess).not.to.be.calledWith(pullAgs)
 
       mockShowInformationMessage.resetHistory()
 
       await commands.executeCommand(openFileCommand, uri)
 
       expect(mockShowInformationMessage).not.to.be.called
-      expect(mockProcess).not.to.be.called
+      expect(mockProcess).not.to.be.calledWith(pullAgs)
     })
 
     it('should be able to run dvc.removeTarget without error', async () => {

